platforms/corda-ent/configuration/samples/network-cordaent.yaml

```diff
@@ -22,6 +22,7 @@ network:
   # Do not check-in docker_password.
   docker:
     url: "index.docker.io/hyperledgerlabs"
+    private_repo: "adopblockchaincloud0502.azurecr.io"
```

Comment:
private_url or private_repo? dont think this is needed as separate as the hyperledgerlabs is public so any image like hyperledgerlabs/alpine_utils will be downloaded automatically